src/App.js

```diff
@@ -1,26 +1,97 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
```

Comment:
"I tried importing it but it never came up auto-fill."
Strangely, .css files don't generally auto-fill in react. They still work though! If you imported as shown here your stylesheet should have been accessible.